pkg/homedir/homedir_unix.go

```diff
@@ -169,7 +172,7 @@ func GetRuntimeDir() (string, error) {
 			if err == nil && int(st.Sys().(*syscall.Stat_t).Uid) == os.Geteuid() && isWriteableOnlyByOwner(st.Mode().Perm()) {
```

Comment:
Did  you forget to remove this isWritableOnlyByOwner check?  Or the error message below should not be changed.